src/default_panels/StyleTracesPanel.js

```diff
@@ -593,13 +593,15 @@ const StyleTracesPanel = (props, {localize: _}) => (
         'scatterpolargl',
       ]}
     >
-      <Flaglist
+      <Dropdown
```

Comment:
why this change?

Reply:
Because there are only three options - you cannot deselect both, I found it kinda funny. Subjective really.